.github/workflows/ci.yml

```diff
@@ -38,9 +38,6 @@ jobs:
       - name: Install dependencies
         run: poetry install
 
-      - name: Install Playwright browsers
```

Comment:
I remember having to add this to make things work, just wondering why it is not needed anymore?

Reply:
I removed it and the ci kept working.